carbon-design-system/carbon
# feat(icons): update glyph icon names

This PR updates the names of glyph assets to remove the `glyph--` prefix. This lead to redundant naming like `GlyphCautionGlyph`.

We have left these `glyph--` prefixed assets in as-is to be removed in v11 and copied them over to the new naming scheme.

#### Changelog

**New**

- Copy over `glyph--` assets to a base name SVG file

**Changed**

- Update `icons.yml` with new assets
- Update `categories.yml` with new categories
- Update `deprecated.yml` with deprecated `glyph--` icons

**Removed**

#### Testing / Reviewing

- Verify all metadata in `icons.yml` and `categories.yml` matches with the changed `glyph--` assets
- Verify all `glyph--` assets are listed in `deprecated.yml` to be removed in v11
